salesforce-connector-demo/src/salesforce/connector/demo/ChartBean.java

```diff
@@ -0,0 +1,154 @@
+package salesforce.connector.demo;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.primefaces.model.charts.ChartData;
+import org.primefaces.model.charts.axes.cartesian.CartesianScales;
+import org.primefaces.model.charts.axes.cartesian.linear.CartesianLinearAxes;
+import org.primefaces.model.charts.axes.cartesian.linear.CartesianLinearTicks;
+import org.primefaces.model.charts.bar.BarChartDataSet;
+import org.primefaces.model.charts.bar.BarChartModel;
+import org.primefaces.model.charts.bar.BarChartOptions;
+import org.primefaces.model.charts.optionconfig.animation.Animation;
+import org.primefaces.model.charts.optionconfig.legend.Legend;
+import org.primefaces.model.charts.optionconfig.legend.LegendLabel;
+import org.primefaces.model.charts.optionconfig.title.Title;
+
+import com.axonivy.connector.salesforce.model.Opportunity;
+
+import ch.ivyteam.ivy.environment.Ivy;
+import salesforce.connector.demo.dto.OpportunityDTO;
+import salesforce.connector.demo.enums.Stage;
+
+public class ChartBean {
+	List<Opportunity> opportunities;
+	List<OpportunityDTO> opps;
+	private BarChartModel barModel;
+
+	public ChartBean() {
+		opps = new ArrayList<>();
+		opportunities = Utils.getAllOpps();
+		Ivy.log().info("list opportunities: " + opportunities.size());
```

Comment:
do we need this log to investigate issues in future? if no, I think we should remove it, @HoDacVu90